test_unstructured/partition/test_auto.py

```diff
@@ -570,17 +568,22 @@ def test_auto_partition_pdf_from_file(pass_metadata_filename: bool, content_type
     assert e.text.startswith("Zejiang Shen")
 
 
-def test_auto_partition_pdf_with_fast_strategy(monkeypatch: MonkeyPatch):
-    file_path = example_doc_path("pdf/layout-parser-paper-fast.pdf")
+def test_auto_partition_pdf_with_fast_strategy(request: FixtureRequest):
+    partition_pdf_ = function_mock(
+        request,
+        "unstructured.partition.pdf.partition_pdf",
+        return_value=[NarrativeText("Hello there!")],
+    )
+    partitioner_loader_get_ = method_mock(
+        request, _PartitionerLoader, "get", return_value=partition_pdf_
+    )
+    filename = example_doc_path("pdf/layout-parser-paper-fast.pdf")
```

Comment:
Wouldn't this be `file_path`? I thought `filename` would be `"layout-parser-paper-fast.pdf"`

Reply:
Yep, good call.